### PR TITLE
[Application] Fix Resnet18 @open sesame 06/13 10:05

### PR DESCRIPTION
**Pull request by SeoHyungjun**
Fix it because the results of the computation of pytorch and nntrainer are different.

padding was written as same when stride was 2 in NNtrainer Resnet code. In pytorch, this is set to error. 
In addition, padding was not applied normally in nntrainer, so the results were different. 

To solve this problem, the parameters of a1 layer (conv layer) have been modified.

Additionally, momentum and epsilon were added to batch_norm layer.

### CODE REVIEW

**Review by myungjoo — APPROVED** [review]

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Review by baek2sm — APPROVED** [review]
LGTM

**Review by DonghakPark — APPROVED** [review]
LGTM!

**Review by DonghakPark — COMMENTED** [review]
Please Check CI in Tizen&Ubuntu Build
